src/renderers/native/ReactNativeTypes.js

```diff
@@ -53,6 +53,19 @@ export type NativeMethodsMixinType = {
   setNativeProps(nativeProps: Object): void,
 };
 
+export interface IReactNativeComponent {
+  blur(): void,
+  focus(): void,
+  measure(callback: MeasureOnSuccessCallback): void,
+  measureInWindow(callback: MeasureInWindowOnSuccessCallback): void,
+  measureLayout(
+    relativeToNativeNode: number,
+    onSuccess: MeasureLayoutOnSuccessCallback,
+    onFail: () => void,
+  ): void,
+  setNativeProps(nativeProps: Object): void,
+}
```

Comment:
I'll change `ReactNativeComponent` to use `NativeMethodsMixinType` like you did in `ReactNativeFiberHostComponent`, but disclaimer... the same trick does not actually work for `ReactNativeComponent`:

```
(ReactNativeComponent.prototype: NativeMethodsMixinType);
```

Unlike for `ReactNativeFiberHostComponent`, Flow does not complain if any of the methods are missing. I suspect this is because `ReactBaseClasses` (and therefore `React.Component`) is not Flow-typed. Getting rid of `extends React.Component` fixes this, but we obviously cannot do that.